docs/sources/mimir/operators-guide/mimir-runbooks/_index.md

```diff
@@ -592,18 +592,37 @@ How to **investigate**:
-      - This is caused by a bug in the ingester - see [mimir#1537](https://github.com/grafana/mimir/issues/1537). Ingesters upload blocks where the MinT and MaxT of some chunks don't match the first and last samples in the chunk. When the faulty chunks' MinT and MaxT overlap with other chunks, the compactor merges the chunks. Because one chunk's MinT and MaxT are incorrect the merge may be performed incorrectly, leading to OoO samples.
-      - **How to mitigate**: Mark the faulty blocks to avoid compacting them in the future:
-        - Find all affected compaction groups in the compactor logs. You will find them as `invalid result block /data/compact/<compaction_group>/<result_block>`.
-        - For each failed compaction job
-          - Pick one result block (doesn't matter which)
-          - Find source blocks for the compaction job: search for `msg="compact blocks"` and a mention of the result block ID.
-          - Mark the source blocks for no compaction (in this example the object storage backend is GCS):
-            ```
-            ./tools/markblocks/markblocks -backend gcs -gcs.bucket-name <bucket> -mark no-compact -tenant <tenant-id> -details "Leading to out-of-order chunks when compacting with other blocks" <block-1> <block-2>...
-            ```
+  - Out-of-order chunks
+    - **How to detect**: Search compactor logs for `invalid result block` and `out-of-order chunks`.
+    - This is caused by a bug in the ingester - see [mimir#1537](https://github.com/grafana/mimir/issues/1537). Ingesters upload blocks where the MinT and MaxT of some chunks don't match the first and last samples in the chunk. When the faulty chunks' MinT and MaxT overlap with other chunks, the compactor merges the chunks. Because one chunk's MinT and MaxT are incorrect the merge may be performed incorrectly, leading to OoO samples.
+    - **How to mitigate**: Mark the faulty blocks to avoid compacting them in the future:
+      - Find all affected compaction groups in the compactor logs. You will find them as `invalid result block /data/compact/<compaction_group>/<result_block>`.
+      - For each failed compaction job
+        - Pick one result block (doesn't matter which)
+        - Find source blocks for the compaction job: search for `msg="compact blocks"` and a mention of the result block ID.
+        - Mark the source blocks for no compaction (in this example the object storage backend is GCS):
+          ```
+          ./tools/markblocks/markblocks -backend gcs -gcs.bucket-name <bucket> -mark no-compact -tenant <tenant-id> -details "Leading to out-of-order chunks when compacting with other blocks" <block-1> <block-2>...
+          ```
+
+Another possible cause of this alert is if compaction has never been running successfully, potentially for a long time. This can be because the alert either was not seen, misconfigured or not installed at all for some time.
```

Comment:
That's a fair point. I was thinking from the perspective of a recent issue seen, but you're right it could happen for any alerts, though this particular one's impact can take longer to recover from, but it's probably clearer to just remove it. As long as people can search the alert and find this section they'll find their answer.